examples/inference/api_server_openai/query_openai_sdk.py

```diff
@@ -40,6 +40,12 @@
     help="If set to float < 1, only the smallest set of most probable tokens with probabilities that add up to`Top p` or higher are kept for generation",
 )
 
+parser.add_argument(
```

Comment:
I think the debug mode should be enabled only from server itself, otherwise if client side can control what the server print would introduce security issues in production.

Reply:
There are two configs for both server side and client side. One is in serve.py for server side and the other is in query_openai_sdk.py for client side.

Reply:
client side are examples for end users to use, there is no need to add debug for them. our primary deliverable is the server side.